source/frontend/src/interfaces/editable-contact/index.ts

```diff
@@ -46,8 +46,10 @@ export interface IBaseAddress {
   regionId?: number;
   districtId?: number;
   provinceId?: number;
+  province?: string;
   countryId?: number;
   countryOther?: string;
+  country?: string;
   postal?: string;
```

Comment:
I guess this is okay but this is an example of why we would benefit from using "concept" models instead. 
Did you check that this change doesn't break current functionality in Contacts? (as those models are used only there?)

Reply:
removed